en/README.md

```diff
@@ -0,0 +1,1138 @@
+|count    |Number   |M|-|Event count of getting.|
+|direction|String255|M|-|Select "ASC"(newer event than designate ID) or $B!I(BDESC$B!I(B(older event that designate ID).|
+|fetchId  |String255|M|-|Use in putEvents procedures. It need to identify the putEvents corresponding to the fetchEvents.|
+
+```json
+{
+  "id": 1,
+  "params": {
+    "direction": "ASC",
+    "count": 1000,
+    "lastInfo": "10",
+    "fetchId": "1"
+  },
+  "method": "fetchEvents",
+  "jsonrpc": "2.0"
+}
+```
+
+***Result(result)***
+
+Return result as result object value whether request accept to Hatohol server. Look at the [[Values](#user-content-putresult)].
+
+```json
+{
+  "id": 1,
+  "result": "SUCCESS",
+  "jsonrpc": "2.0"
+}
+```
+
+### updateMonitoringServerInfo(notification)
+
+ This procedure is sent from Hatohol server when updated monitoring server information. At that time, should update each monitoring server information to restart HAP.
+
+***params***
+
+params content is same as the result object of [getMonitoringServerInfo](#user-content-getmonitoringserverinfomethod).
+
+```json
+{
+  "method": "updateMonitoringServerInfo",
+  "params": {
+    "extendedInfo": "exampleExtraInfo",
+    "serverId": 1,
+    "url": "http://example.com:80",
+    "type": "12345678-9abc-def0-1234-567891abcdef",
+    "nickName": "exampleName",
+    "userName": "Admin",
+    "password": "examplePass",
+    "pollingIntervalSec": 30,
+    "retryIntervalSec": 10
+  },
+  "jsonrpc": "2.0"
+}
+```
+
+## Tables
+
+### armInfoStatus
+
+|Status|Description|
+|:---------|:---|
+|"INIT"   |Initial status. Not communication yes.|
+|"OK"     |Succeed connection.|
+|"NG"     |Fail connection.|
+
+### ServerType
+
+If you create a HAP, need to define new server type that has UUID other than the following UUIDs.
+
+|$BL>A0(B|UUID|
+|:---|:---|
+|Zabbix    |8e632c14-d1f7-11e4-8350-d43d7e3146fb|
+|Nagios    |902d955c-d1f7-11e4-80f9-d43d7e3146fb|
+|Ceilometer|aa25a332-d1f7-11e4-80b4-d43d7e3146fb|
+
+### triggerSeverity
+
+|Severity|Description|
+|:---|:--|
+|"UNKNOWN"  |Unknown|
+|"INFO"     |Information|
+|"WARNING"  |Warning|
+|"ERROR"    |Error|
+|"CRITICAL" |Critical|
+|"EMERGENCY"|Emergency|
+
+### triggerStatus
+
+|Status|Description|
+|:---------|:---|
+|"OK""    |Succeed connection.|
+|"NG"     |Fail connection.|
+|"UNKNOWN"|Status is unknown|
+
+### updateType
+
+|Type|Description|
+|:---------|:---|
+|"ALL"    |Send all data. Remove old data from hatohol server and regist all sent data.|
+|"UPDATED"|Send only updata data, after that If data ID already exists in Hatohol server, overwrite data and in case of does not exist, initial regist that data.|
+
+### eventType
+
+|Type|Description|
+|:---|:---------:|
+|"GOOD"        |Normal|
+|"BAD"         |Abnormal|
+|"UNKNOWN"     |Unknown|
+|"NOTIFICATION"|Notification|
+
+### putResult
+
+The following table value are success and failure of to call put procedure. If fail to update database, to recall procedure is standard motion.
```

Comment:
> The following table value are success and failure of to call put procedure.

The following table value**s** represents whether success or failure when calling put procedure series.

>  If fail to update database, to recall procedure is standard motion.

If it fails to update database, recalling procedure is standard behavior.